<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_e">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>341</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>공동현관</string>
   </property>
   <widget class="QLineEdit" name="le_e_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_e_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_e_approv">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ele">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>엘리베이터</string>
   </property>
   <widget class="QLCDNumber" name="lcdNumber_floor">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>100</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>층</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1층</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2층</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>3층</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ele_3f">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ele_2f">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ele_1f">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>16</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>✅</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>411</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Home</string>
   </property>
   <widget class="QWidget" name="widget_graph" native="true">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>50</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_temp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_hu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>온도(C)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>습도(%)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_air">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>에어컨 ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_heat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>히터 ON/OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_humi">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>가습기 ON/OFF</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_lux">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>외부조도(LUX)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curOpen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>커튼 OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curClose">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>커튼 CLOSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_curStop">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>커튼 STOP</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_cur">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>커튼 열림 정도</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
